### ISSUE TITLE
Cheating is coming up.

### ISSUE THREAD

**Issue by devashish2024** [Digitso/Digitso-Buttons#2]
You were are using the btns.css framework, I shared in @Educational-Websites. You were are trying to make it yourself, Doesn't mean that you can make same styles so it's yours. If it's same, a case will come up.

**Comment by Sreehari521** [comment]
> You were are using the btns.css framework, I shared in @Educational-Websites. You were are trying to make it yourself, Doesn't mean that you can make same styles so it's yours. If it's same, a case will come up.

No. It's not the code of btns.css. Anyone can create on like this their self. Everyone add these types of Buttons in their Website

**Comment by Sreehari521** [comment]
> You were are using the btns.css framework, I shared in @Educational-Websites. You were are trying to make it yourself, Doesn't mean that you can make same styles so it's yours. If it's same, a case will come up.

I am going to give the Idea & Inspiration credit to btns.css

**Comment by devashish2024** [comment]
But, If all codes output become same, It is called copying.

**Comment by devashish2024** [comment]
It's btns.css buttons, of btns.css. Not our. Remove it and make your own for digistio.

**Comment by Sreehari521** [comment]
> It's btns.css buttons, of btns.css. Not our. Remove it and make your own for digistio.

Then can u create for digitso?

**Comment by devashish2024** [comment]
No, But Digistio is yours.

**Comment by Sreehari521** [comment]
> No, But Digistio is yours.

ok ok , I will try

**Comment by devashish2024** [comment]
Before trying, open this issue again for a proof.

**Comment by Super45coder** [comment]
Okay. Maybe this should be a discussion.

**Comment by devashish2024** [comment]
No. Not needed. There's the proof. NO CLOSE

**Comment by Sreehari521** [comment]
> No. Not needed. There's the proof. NO CLOSE

Come to google hangouts @e-coders

**Comment by devashish2024** [comment]
sure
